PWGHF/D2H/DataModel/ReducedDataModel.h

```diff
@@ -183,16 +163,66 @@ auto invMassDplusToPiKPi(const T& pVec0, const T& pVec1, const T& pVec2)
                                  RecoDecay::getMassPDG(kKPlus),
                                  RecoDecay::getMassPDG(kPiPlus)});
 }
-} // namespace hf_cand_3prong_reduced
+} // namespace hf_charm_cand_reduced
 
-DECLARE_SOA_TABLE(HfCand3ProngReduced, "AOD", "HFCAND3PRONGRED", //! Table with 3prong candidate information for reduced workflow
+// CAREFUL: need to follow convention [Name = Description + 's'] in DECLARE_SOA_TABLE(Name, "AOD", Description)
+// to call DECLARE_SOA_INDEX_COLUMN_FULL later on
+DECLARE_SOA_TABLE(HfRed2Prongs, "AOD", "HFRED2PRONG", //! Table with 2prong candidate information for reduced workflow
+                  o2::soa::Index<>,
+                  hf_track_index_reduced::Prong0Id, hf_track_index_reduced::Prong1Id,
+                  hf_track_index_reduced::HfRedCollisionId,
+                  HFTRACKPAR_COLUMNS,
+                  hf_cand::XSecondaryVertex, hf_cand::YSecondaryVertex, hf_cand::ZSecondaryVertex,
+                  hf_charm_cand_reduced::InvMassD0, hf_charm_cand_reduced::InvMassD0Bar,
+                  aod::track::Px<aod::track::Signed1Pt, aod::track::Snp, aod::track::Alpha>,
+                  aod::track::Py<aod::track::Signed1Pt, aod::track::Snp, aod::track::Alpha>,
+                  aod::track::Pz<aod::track::Signed1Pt, track::Tgl>);
+
+DECLARE_SOA_TABLE(HfRed2ProngsCov, "AOD", "HFRED2PRONGSCOV", //! Table with 2prong candidate covariance for reduced workflow
                   o2::soa::Index<>,
-                  hf_track_index::Prong0Id, hf_track_index::Prong1Id, hf_track_index::Prong2Id,
-                  hf_track_index_reduced::HfReducedCollisionId,
                   HFTRACKPARCOV_COLUMNS,
-                  hf_cand_3prong_reduced::CPA,
-                  hf_cand_3prong_reduced::DecayLength,
-                  hf_cand_3prong_reduced::InvMass);
+                  o2::soa::Marker<1>);
```

Comment:
can you just remind me what this `Marker<1>` means?

Reply:
Hi @mfaggin , this `Marker<>` allows to define several tables with the same columns. As we want to save the same information in `HfRed2ProngsCov` and `HfRed3ProngsCov`, but we still want to distinguish them by name and description, we mark the 2prongs table with `Marker<1>` and the 3prongs one with `Marker<2>`.

Reply:
thank you!